<launch>
  <include file="$(find sapien_robot_moveit_config)/launch/move_group.launch">
  </include>
  <node name="point_cloud_merger" pkg="motion_planning" type="depth_to_pcd_merger.py">
  </node>
</launch>
